peer/brontide.go

```diff
@@ -403,10 +404,6 @@ type Brontide struct {
 	// objects to queue messages to be sent out on the wire.
 	outgoingQueue chan outgoingMsg
 
-	// activeChanMtx protects access to the activeChannels and
-	// addedChannels maps.
-	activeChanMtx sync.RWMutex
```

Comment:
IIRC, the active channels map was originally only ever accessed in the main event loop. Once we added `filterChannelsToEnable` to handle channel enable/disable, then we created that other `addedChannels` map to track the _new_ channels since the peer session was created. 

These maps are also only ever modified right after each other in the main event loop `channelManager`.

Reply:
If we wanted to, we could get rid of _both_ these mutexes if we funnel all the state requests through the main `channelManager` goroutine (req+resp over channels).